tests/test_api_raw.py

```diff
@@ -0,0 +1,44 @@
+from .apitest_shared import *
```

Comment:
please use `from tests.apitest_shared import *`. This avoids issues with relative file imports that can occur when you call your test code from another directory.